### PR TITLE
3.6 release

### PR DESCRIPTION
**Pull request by pascalchevrel**
* Make News page use MVC
* Add helper functions in model for News page to make updates easier

### CODE REVIEW

File: app/inc/dispatcher.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
         $page_descr = 'One source locale, get search results for two target locales';
         break;
     case 'news':
-        $view = 'changelog';
+        $controller = 'changelog';
         $page_title = 'Transvision News. Version Notes';
```
Comment (flodolo) [inline app/inc/dispatcher.php]: Never really thought about this title. Maybe `Transvision News and Release Notes`?

Comment (pascalchevrel) [inline app/inc/dispatcher.php]: Sounds good,will change that in a minute

File: app/models/changelog.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        default:
+            $type = '';
+            $text = 'Other';
+            break;
+    }
+    return "<span class=\"release_tag {$type}\">{$text}</span> ";
+};
+
+// Helper to generate the list of patches for the version
+$github_link = function($release) use($releases) {
```
Comment (flodolo) [inline app/models/changelog.php]: We're not exactly consistent at the moment, but most of the code is using `function(vars) use (vars)` with a space after `use` (there are 2 cases without space). Same below.

File: app/models/changelog.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    $keys = array_keys($releases);
+    $previous = $keys[array_search($release, $keys) -1];
```
Comment (flodolo) [inline app/models/changelog.php]: nit: space before `1`

File: app/views/changelog.php [hunk 4]
```diff
@@ -1,50 +1,80 @@
...
-?>
-<h2 class="release_number" id="v3.5.1"><a href="#v3.5.1">Version 3.5.1 <span class="release_date">2014-10-07</span></a></h2>
-
+<?=$release_title('3.6')?>
 <h3>End user visible changes</h3>
 <ul>
-    <li><?=$relnotes('bug')?>Fixed a bug that prevented new Gaia repositories – fetched from external web service – to appear in the list of available repositories when performing a search (flod)</li>
+    <li>
+        <?=$relnotes('new')?>
+        Added a new view exposing all <a href="/unchanged/">strings that were kept identical to en-US</a> (flod)
```
Comment (flodolo) [inline app/views/changelog.php]: were kept -> are kept

File: app/views/changelog.php [hunk 5]
```diff
@@ -1,50 +1,80 @@
...
+        <?=$issue(413)?>
+        Replace Kint library by Symfony/var_dumper library for quick debugging, new <code>dump()</code> and <code>cli_dump()</code> helper functions (pascal)
+    </li>
+    <li>
+        <?=$relnotes('bug')?>
+        Archlinux compatibility fix: Force the use of Python 2 and not 3 (SkySymbol)
+    </li>
+    <li>
+        <?=$relnotes('change')?>
+        Bugzilla and  Cache classes were moved as external libraries for sharing with other l10n-drivers PHP tools (flod)
```
Comment (flodolo) [inline app/views/changelog.php]: not really used, but double space before Cache

File: app/models/changelog.php (hunk 2 above)
Comment (pascalchevrel) [inline app/models/changelog.php]: Just checked the PSR recommandation and they advice adding spaces:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#6-closures
"Closures MUST be declared with a space after the function keyword, and a space before and after the use keyword."

**Comment by flodolo** [comment]
I would add #390, maybe in the API section even if it affects also website queries. 
Example: "Queries for mozilla.org don't return English text when the string is untranslated". 

We lost a section for 3.5.1: is that wanted?

File: app/models/changelog.php (hunk 2 above)
Comment (flodolo) [inline app/models/changelog.php]: Interesting. We're definitely missing a space after `function` in a ton of places

**Pull request merged**